packages/x-scheduler/src/joy/models/events.ts

```diff
@@ -0,0 +1,20 @@
+import { SchedulerValidDate } from '../../primitives/utils/adapter/types';
+
+export interface CalendarEvent {
+  /**
+   * The unique identifier of the event.
+   */
+  id: string;
```

Comment:
We will probably add a `getEventId` like the `getItemId` of the tree view and the `getRowId` of the grid, but I'd prefer to wait before introducing this kind of abstraction

Reply:
Makes sense. 👍